eZ/Bundle/EzPublishCoreBundle/Tests/Translation/CollectorTest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * File containing the CollectorTest class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Bundle\EzPublishCoreBundle\Tests\Translation;
+
+use eZ\Bundle\EzPublishCoreBundle\Translation\Collector;
+
+class CollectorTest extends \PHPUnit_Framework_TestCase
+{
+    public function testCollect()
+    {
+        $translationRootDir = str_replace('/Tests/Translation', '/Tests/Resources/Translation/vendor', __DIR__);
```

Comment:
@nicolas-bastien if you need to hardcode filesystem path, please use `DIRECTORY_SEPARATOR` PHP constant. That's why it fails on AppVeyor. 
But as @bdunogier suggests, vfs might be a better option in this case :)